src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
> Technically they're part of the javaRosa plugin, but that plugin is so tightly coupled with the rest of vellum that I think it makes sense to include them here

side note: Should we stop assuming that someone could use vellum without javaRosa? We could get rid of label and hintLabel at least, maybe some other things?